### PR TITLE
Implement `WLF` in `BhpQc`

### LINKED ISSUE #1072: `BhpQc` to WLF
https://webviz-subsurface-example.azurewebsites.net/bhp-qc

### PR DESCRIPTION
**Pull request by rnyb**
Based on  BhpQc to WLF #1079. Created new branch/PR to work around merge conflicts with original PR.

---

### Contributor checklist

- [x] :tada: This PR closes #1072.
- [ ] :scroll: I have broken down my PR into the following tasks:
   - [ ] Task 1
   - [ ] Task 2
- [ ] :robot: I have added tests, or extended existing tests, to cover any new features or bugs fixed in this PR.
- [x] :book: I have considered adding a new entry in `CHANGELOG.md`, and added it if should be communicated there.

### CODE REVIEW

File: webviz_subsurface/plugins/_bhp_qc/_plugin.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+                rel_file_pattern,
+            )
+        except PermissionError:
+            self.error_message = f"Access to files '{rel_file_pattern}' denied."
+            "Please check your path for 'rel_file_pattern'"
+            return
+        except FileNotFoundError:
+            self.error_message = f"Files '{rel_file_pattern}' not found."
+            "Please check your path for 'rel_file_pattern'."
+            return
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_bhp_qc/_plugin.py:71]: Why do we not want to fail in these two cases? 🙂

**Review by anders-kiaer — COMMENTED** [review]
Nice. LGTM! 👍🚀  I have only one small question then this looks ready to me.

File: webviz_subsurface/plugins/_bhp_qc/_plugin.py (hunk 1 above)
Comment (rnyb) [reply webviz_subsurface/plugins/_bhp_qc/_plugin.py]: Removed the exceptions all-together as they are already handled implicitly by `create_lazy_provider_set_from_paths`

**Review by anders-kiaer — APPROVED** [review]
LGTM 👍 Ready for merge once the tests are 🟢

**Pull request merged**